webextensions/api/webNavigation.json

```diff
@@ -414,6 +414,101 @@
               }
             }
           },
+          "documentId": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "106"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "documentLifecycle": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "106"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "frameType": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "106"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "parentDocumentId": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "106"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "parentFrameId": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "106"
```

Comment:
Is this value correct? Chrome's API documentation indicates that this property was added in Chrome 74. https://developer.chrome.com/docs/extensions/reference/api/webNavigation#property-onCommitted-callback-details-parentFrameId

It also looks like this property is exposed on nine Web Navigation API events in total.

Reply:
That's annoying, because I knew that (as you see, I even commented on it in the summary information for the PR). Exposure in the other events, I did overlook. However, only introduced in 74 in onCompleted, onDOMContentLoaded, onErrorOccurred, onHistoryStateUpdated, and onReferenceFragmentUpdated, and those are now added.